plugins/core/techleads.go

```diff
@@ -0,0 +1,33 @@
+package core
+
+import (
+	"github.com/nlopes/slack"
+	log "github.com/sirupsen/logrus"
+	"github.com/ubclaunchpad/rocket/cmd"
+	"github.com/ubclaunchpad/rocket/model"
+)
+
+// NewTechLeadsCmd returns a teams command that displays a list of Launch Pad teams
+func NewTechLeadsCmd(ch cmd.CommandHandler) *cmd.Command {
+	return &cmd.Command{
+		Name:       "admins",
+		HelpText:   "List Launch Pad admins",
```

Comment:
I think you forgot to change `admins` to `tech leads` here when you copy-pasted.